### PR TITLE
ramips: Add support for ASUS RT-AC51U / RT-AC54U

### PR DESCRIPTION
**Pull request by p055877632**
[https://github.com/openwrt/openwrt/pull/132](https://github.com/openwrt/openwrt/pull/132)

The previous version did not complete

I am so sorry

### CODE REVIEW

**Comment by p055877632** [comment]
p.s.

power2 {
label = "rt-ac51u:blue:power2";
gpios = <&gpio0 10 0>;

is ALL LED Control  FOR
5G WIFI LED (mt76 not support)
2.4G WIFI LED
USB LED
WAN LED
LAN LED

don't  remove

**Comment by p055877632** [comment]
gpio10 LED trap......

Asus and odm Factory edimax/acelink group "thanks a lot"

f*** you

File: target/linux/ramips/base-files/lib/ramips.sh [hunk 1]
```diff
@@ -373,6 +373,12 @@ ramips_board_detect() {
 	*"RT-N14U")
 		name="rt-n14u"
 		;;
+	*"RT-AC54U")
+		name="rt-ac54u"
+		;;
+	*"RT-AC51U")
+		name="rt-ac51u"
+		;;
 	*"RT-N15")
```
Comment (blogic) [inline target/linux/ramips/base-files/lib/ramips.sh]: please stick to alphabetical ordering

File: target/linux/ramips/base-files/lib/upgrade/platform.sh [hunk 2]
```diff
@@ -109,6 +109,8 @@ platform_check_image() {
 	rt-n10-plus|\
 	rt-n13u|\
 	rt-n14u|\
+	rt-ac54u|\
+	rt-ac51u|\
 	rt-n15|\
```
Comment (blogic) [inline target/linux/ramips/base-files/lib/upgrade/platform.sh]: please stick to alphabetical ordering

**Review by blogic — CHANGES_REQUESTED** [review]
the two dts files look almost identical. please consider merging them into a dtsi file and including this in botht he dts files.

File: target/linux/ramips/base-files/lib/upgrade/platform.sh (hunk 2 above)
Comment (p055877632) [inline target/linux/ramips/base-files/lib/upgrade/platform.sh]: NO

AC51U 
AC54U

This is 2 Router
(maybe one pcb)

Comment (wigyori) [inline target/linux/ramips/base-files/lib/upgrade/platform.sh]: "Yes". If you add separate entries, stick to alphabetical ordering.

**Comment by blogic** [comment]
already part of the LEDE tree which will replace the OWRT tree shortly

**Pull request closed without merging**